src/internal/yey_test.go

```diff
@@ -0,0 +1,89 @@
+package yey
+
+import "testing"
+
+func TestSanitizePathName(t *testing.T) {
+	testCases := []struct {
+		Name     string
+		Value    string
+		Expected string
+	}{
+		{
+			Name:     "removes space from  name",
+			Value:    "my folder with   spaces",
+			Expected: "my_folder_with_spaces",
+		},
+		{
+			Name:     "strips invalid characters",
+			Value:    "my&folder#cool*stuff!",
+			Expected: "myfoldercoolstuff",
+		},
+		{
+			Name:     "padds with 0 if does not start with alphanumeric character",
+			Value:    "_valid_if_not_for_underscore_start",
+			Expected: "0_valid_if_not_for_underscore_start",
+		},
+		{
+			Name:     "reduces multidashes to single dash",
+			Value:    "my---folder----path",
+			Expected: "my-folder-path",
+		},
+		{
+			Name:     "strips trailing dashes from name",
+			Value:    "my_folder----",
+			Expected: "my_folder",
+		},
+		{
+			Name:     "empty string",
+			Value:    "",
+			Expected: "",
+		},
+		{
+			Name:     "full strip returns empty string",
+			Value:    "$%^&",
+			Expected: "",
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.Name, func(t *testing.T) {
+			actual := sanitizePathName(tc.Value)
+			if actual != tc.Expected {
```

Comment:
pourquoi pas:
```
assert.Equals(t, tc.Expected, actual)
```
???

Reply:
absolutely no reason. forgot we had that depedency. Blocker?